corehq/apps/app_manager/app_schemas/tests/test_schema.py

```diff
@@ -122,7 +122,7 @@ def test_get_casedb_schema_with_parent_case_property_update(self):
             "key": "@case_id",
         }})
         self.assertEqual(subsets["case"]["structure"]["case_name"], {"description": ""})
-        #self.assertEqual(subsets["parent"]["structure"]["has_well"], {}) TODO
+        self.assertEqual(subsets["parent"]["structure"]["has_well"], {"description": ""})
```

Comment:
Finally getting to uncomment this line!
https://github.com/dimagi/commcare-hq/commit/33642a7bbf942fb17bbfa652e72c54f16bd0f189#diff-b9de4a695c074e8e8121b6e74f402a9dR168

Reply:
🎉